gemonlinedb/xml/schema/system-topology.xsd

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" standalone="yes"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:gem="https://github.com/cms-gem-daq-project/cmsgemos/tree/master/gemonlinedb/xml/schema/system-topology.xsd"
+           targetNamespace="https://github.com/cms-gem-daq-project/cmsgemos/tree/master/gemonlinedb/xml/schema/system-topology.xsd"
+           elementFormDefault="qualified">
+
+    <!-- Attributes -->
+    <xs:attribute name="serial-number" type="xs:string"/>
+    <xs:attribute name="barcode" type="xs:string"/>
+
+    <!-- Empty slot -->
+    <xs:element name="empty-slot"/>
+
+    <!-- VFAT 3 -->
+    <xs:element name="vfat3">
+        <xs:complexType>
+            <xs:attribute ref="gem:barcode" use="required"/>
+        </xs:complexType>
+    </xs:element>
+
+    <!-- OHv3 -->
+    <xs:element name="ohv3">
+        <xs:complexType>
+            <xs:choice minOccurs="24" maxOccurs="24">
```

Comment:
Could have fewer, depending on the use case, or does the `empty-slot` cover cases when there are fewer actual `VFATs` connected?

Reply:
That's what `<empty-slot>` does, yes. Entries are expected to be sequential. I should probably add documentation comments to the examples.